_rules/iframe-not-focusable-has-no-interactive-content-akn7bn.md

```diff
@@ -28,11 +28,11 @@ acknowledgments:
 
 ## Applicability
 
-This rule applies to any `iframe` element that has a negative number as a `tabindex` [attribute value][].
+This rule applies to any non-focusable `iframe` element that has focusable content.
```

Comment:
I'm also not sure about adding "that has focusable content" in the Applicability. Skipping over the (lack of) definition of "has", this means that most Passed Example (e.g. Passed Example 1) are now Inapplicable, since the `iframe` has no focusable content 🤔